web_sdk/golden_comparator/lib/image_compare.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:image/image.dart';
+import 'package:path/path.dart' as p;
+
+import 'package:common_test_utils/environment.dart';
+import 'goldens.dart';
+
+/// Compares a screenshot taken through a test with it's golden.
+///
+/// Used by Flutter Web Engine unit tests and the integration tests.
```

Comment:
The docs should explain what the returned `String` contains.

Reply:
done. (btw the string values are not added with this PR, they are added when the first tests were in).